Daytime-Nighttime-Interactions/slack-channel-usage.rst

```diff
@@ -0,0 +1,196 @@
+People who participate in testing or infrastructure support/development are expected to be actively monitoring this channel.
+
+Because work is being coordinated across multiple channels, the creation of this channel aims to restrict the need to make announcements multiple places and avoid the scenario of accidentally not informing all impacted parties.
+
+For items having substantial impact, scheduling should first be being performed as part of the `SUMMIT Jira calendar <https://jira.lsstcorp.org/secure/DoItBetterCalendar.jspa>`_ process.
+
+**Announcement Protocol:**
+
+- Announcements are required for all work that may affect the summit control network or its infrastructure.
+  This includes Nublado updates, invasive networking changes, or interactions with any CSC.
+- Utilize threads whenever possible to minimize cross-talk and/or interference between simultaneous discussions
+
+**Announcement Guidelines and Procedure:**
+
+- Make the announcement when ready to begin work.
+  Include and estimate of how long the systems will be in use.
+  Include name of other channel where conversation on the work will take place (if appropriate).
+  List the CSC(s) you plan on using and a 1 or 2-sentence synopsis of what you are doing.
+
+- Announce when work is completed.
+  This should not be buried in a previous thread but be a new message
+  
+- Announcements for testing using other test-stands are to be made on test-stand specific channels, following the same rules.
+  This includes #rubinobs-tucson-teststand and #rubinobs-base-teststand
+
+
+.. examples include:
+.. maintains its current use-case (broadly)
+.. For information that directly affects the current system users
+.. E.g. Resetting LOVE, starting to work with telescopes etc.
+.. Software updates, IT outages, use of generator power
+.. Required information regarding personnel or hardware safety
+.. Inclement weather, (?)
+.. System control handoffs
+.. Observatory State transitions (coming in the future)
+
+
+
+#summit-control-room
+--------------------
+This room is actively monitored by observing specialists and shift managers.
+
+Communications in this room should be minimal.
+This is a single channel for both telescopes which is actively monitored by telescope specialists and anyone controlling the system.
+For maximum efficiency, primary communication should be via Zoom and not in the channel.
+Breakout rooms can be used if multiple simultaneous conversations are required.
+
+This channel is meant to be used as the area where the copying/pasting of code snippets happens while supporting via zoom, therefore context will often not be provided.
+Troubleshooting in this area should be either system-wide, or issues which benefit from having multiple people see and understand the diagnosis.
+
+An example of troubleshooting belonging to this channel is found in `this Slack snippet <_static/dialog_example.png>`_.
```

Comment:
This was a broken link for me, not sure why.

Reply:
Ya, I hit this the other day with another doc. I don't know why it fails. I pinged jonathan on #rubinobs-sitcom-docs

Reply:
Resolved by dropping the image in-line with the text.